src/lib/components/Header/Title/Title.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  import "./Title.scss";
+  import ldafLogo from "$lib/assets/ldaf-flat-logo-transparent.png";
+  import type { SiteTitleType } from "./types";
+  export let siteTitle: SiteTitleType;
+</script>
+
+<div class="usa-logo ldaf-logo__wide" id="basic-logo">
+  <a href="/" title="<Project title>">
```

Comment:
We should either add a `TODO` here so we don't forget to replace this, or create a `title` that we can throw into `Title.server.ts`. Based on [this a11y guidance](https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo) and since the rest of the descriptive text is not part of the link, I'm thinking we should set the `title` here to something like `"Louisiana Department of Agriculture and Forestry Home"`.

Reply:
Based on that [first example](https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo), does it make sense to just forgo the `title` entirely and just update the `alt` attribute on the `img`? I'm out of my element here, so I don't know what the best practice is.

Reply:
Ah yep, good call. +1 to remove the `title` from the `<a>` and add an `alt` to the `<img>`.

Reply:
to your point, though, the [compact title below this](https://github.com/la-ldaf/ldaf-site/pull/50/files#diff-0f06d8a7ccb1b7bf0eb8bb2cdce882d3e36e483c6fbc8b32b3d641f46a0ae9a9R26) could probably still have a title value served from the CMS